docs/conf.py

```diff
@@ -107,6 +107,13 @@
     },
 }
 
+html_sidebars = {
+    "contributing": [
```

Comment:
The contributing top-level page/directory doesn't exist anymore, so this has no effect?

Reply:
good point - I updated this in the latest commit